content/en/docs/reference/glossary/qos-class.md

```diff
@@ -0,0 +1,24 @@
+---
+title: QoS Class
+id: qos
+date: 2019-04-15
+full_link: 
+short_description: >
+  QoS Classes provides a way for kubernetes to classify pods within the cluster and make decisions about scheduling and eviction.
+
+aka: 
+tags:
+- core-object
+- fundamental
+- architecture
+related:
+ - pod
+
+---
+ QoS Classes provides a way for kubernetes to classify Pods within the cluster and make decisions about scheduling and eviction.
```

Comment:
In the glossary, I suggest that we explain “QoS” (which is jargon that a Kubernetes user might not have encountered), rather than “QoS class”.
The glossary entry for QoS could signpost readers to a page about the _concept_ of a QoS class.

How does that sound?

Reply:
Sounds good @sftim , will update the filename and content accordingly.